### PR TITLE
Update fr.m3u

### PR DESCRIPTION
**Pull request by rphlmm**

### CODE REVIEW

File: streams/fr.m3u [hunk 1]
```diff
@@ -45,10 +47,22 @@ https://edge.vedge.infomaniak.com/livecast/ik:dbmtv/manifest.m3u8
 https://dbmtv.vedge.infomaniak.com/livecast/dbmtv/playlist.m3u8
 #EXTINF:-1 tvg-id="EreTV.fr",Ère TV (1080p)
 https://mn-nl.mncdn.com/awraastv/awraastv_hd.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Eurosport 1",Eurosport 1 (1080p)
+http://145.239.5.177:8080/529/index.m3u8
```
Comment (Dum4G) [inline streams/fr.m3u:51]: DMCA, we can't add it into playlist

File: streams/fr.m3u [hunk 2]
```diff
@@ -45,10 +47,22 @@ https://edge.vedge.infomaniak.com/livecast/ik:dbmtv/manifest.m3u8
...
+#EXTINF:-1 tvg-id="Eurosport 2",Eurosport 2 (720p)
+http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/9280.m3u8
```
Comment (Dum4G) [inline streams/fr.m3u:53]: DMCA, we can't add it into playlist

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Comment by GuyInCanada** [comment]
the http://ipro.webvps.de streams are from an Xtream Codes server.

File: streams/fr.m3u [hunk 3]
```diff
@@ -13,6 +13,8 @@ https://live.creacast.com/albi-tv-ch1/stream/playlist.m3u8
 https://edge.vedge.infomaniak.com/livecast/ik:adhtv/chunklist.m3u8
 #EXTINF:-1 tvg-id="AntenneReunion.fr",Antenne Réunion (720p) [Not 24/7]
 https://live-antenne-reunion.zeop.tv/live/c3eds/antreunihd/hls_fta/antreunihd.m3u8?location=ZEOP01
+#EXTINF:-1 tvg-id="Arte",Arte (720p)
```
Comment (Dum4G) [inline streams/fr.m3u:16]:
```suggestion
#EXTINF:-1 tvg-id="arte.fr",Arte (720p) [Geo-blocked]
```

File: streams/fr.m3u [hunk 4]
```diff
@@ -47,8 +49,16 @@ https://dbmtv.vedge.infomaniak.com/livecast/dbmtv/playlist.m3u8
 https://mn-nl.mncdn.com/awraastv/awraastv_hd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="FashionTVCzechSlovak.fr",FashionTV Czech&Slovak (450p) [Not 24/7]
 http://lb.streaming.sk/fashiontv/stream/playlist.m3u8
+#EXTINF:-1 tvg-id="France 2", France 2 (720p)
+http://tv.pro-ott.com:25443/live/HAJG6/k7bKPqg05y/15226.m3u8
```
Comment (Dum4G) [inline streams/fr.m3u:53]: x-stream codes links are not allowed

File: streams/fr.m3u [hunk 5]
```diff
@@ -47,8 +49,16 @@ https://dbmtv.vedge.infomaniak.com/livecast/dbmtv/playlist.m3u8
...
 #EXTINF:-1 tvg-id="France24English.fr",France 24 English (1080p)
 http://n1.klowdtv.net/live2/france24_720p/chunks.m3u8
+#EXTINF:-1 tvg-id="France 3", France 3 (540p)
+http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/271306.m3u8
```
Comment (Dum4G) [inline streams/fr.m3u:57]: x-stream codes links are not allowed

File: streams/fr.m3u [hunk 6]
```diff
@@ -47,8 +49,16 @@ https://dbmtv.vedge.infomaniak.com/livecast/dbmtv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="France 5", France 5 (720p)
+http://tv.pro-ott.com:25443/live/HAJG6/k7bKPqg05y/15167.m3u8
```
Comment (Dum4G) [inline streams/fr.m3u:59]: x-stream codes links are not allowed

File: streams/fr.m3u [hunk 7]
```diff
@@ -47,8 +49,16 @@ https://dbmtv.vedge.infomaniak.com/livecast/dbmtv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="France TV Info", France TV Info (1080p)
+http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/43444.m3u8
```
Comment (Dum4G) [inline streams/fr.m3u:61]: x-stream codes links are not allowed

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/fr.m3u (hunk 4 above)
Comment (BellezaEmporium) [reply streams/fr.m3u:53]: image

File: streams/fr.m3u (hunk 5 above)
Comment (BellezaEmporium) [reply streams/fr.m3u:57]: image

**Comment by Sphinxroot** [comment]
too sketchy  closing

**Pull request closed without merging**